### PR TITLE
refactor: remove mqtt from ui

### PR DESCRIPTION
**Pull request by rsdmike**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { KvmComponent } from './devices/kvm/kvm.component'
 import { SolComponent } from './devices/sol/sol.component'
 import { DomainDetailComponent } from './domains/domain-detail/domain-detail.component'
 import { DomainsComponent } from './domains/domains.component'
-import { EventChannelComponent } from './event-channel/event-channel.component'
+// import { EventChannelComponent } from './event-channel/event-channel.component'
```
Comment (Walt-Intel) [inline src/app/app-routing.module.ts:18]: Do you want to remove the commented line?

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -59,8 +59,8 @@ const routes: Routes = [
       { path: '', component: WirelessDetailComponent }
     ]
   },
-  { path: 'wireless/:name', component: WirelessDetailComponent },
-  { path: 'event-channel', component: EventChannelComponent }
+  { path: 'wireless/:name', component: WirelessDetailComponent }
+  // { path: 'event-channel', component: EventChannelComponent }
```
Comment (Walt-Intel) [inline src/app/app-routing.module.ts:63]: Do you want to remove the commented line?

File: src/app/app.component.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@
 import { AfterViewInit, Component, OnDestroy, OnInit } from '@angular/core'
 import { NavigationStart, Router } from '@angular/router'
 import { AuthService } from './auth.service'
-import { MQTTService } from './event-channel/event-channel.service'
+// import { MQTTService } from './event-channel/event-channel.service'
```
Comment (Walt-Intel) [inline src/app/app.component.ts:9]: Do you want to remove the commented line?

File: src/app/app.component.ts [hunk 4]
```diff
@@ -16,11 +16,10 @@ import { MQTTService } from './event-channel/event-channel.service'
 export class AppComponent implements OnInit, AfterViewInit, OnDestroy {
   isLoggedIn = false
 
-  constructor (public router: Router, public authService: AuthService, public mqttService: MQTTService
-  ) {
-    this.mqttService.connect()
-    this.mqttService.subscribeToTopic('mps/#')
-    this.mqttService.subscribeToTopic('rps/#')
+  constructor (public router: Router, public authService: AuthService) {
```
Comment (Walt-Intel) [inline src/app/app.component.ts:19]: Do you want to remove the commented lines?

File: src/environments/environment.ts [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
 export const environment = {
   production: false,
-  mpsServer: 'http://localhost:3000',
+  mpsServer: 'https://192.168.1.34/mps',
```
Comment (Walt-Intel) [inline src/environments/environment.ts:12]: Change back to 'localhost'?

File: src/app/app-routing.module.ts (hunk 2 above)
Comment (rsdmike) [reply src/app/app-routing.module.ts:63]: Not particularly, normally, yes. However, i've essentially commented out this entire feature until `ngx-mqtt` can address the CVE. Until then, ripping it out.

**Review by Walt-Intel — APPROVED** [review]
Looks good to me.

**Review by matt-primrose — APPROVED** [review]

**Pull request merged**